### PR TITLE
feat: update contract messages

### PR DESCRIPTION
**Pull request by jhlee-young**

### CODE REVIEW

**Review by nowooj — APPROVED** [review]

File: content/en/docs/how-to/frequently-used-messages.md [hunk 1]
```diff
@@ -62,7 +64,27 @@ In `send.msg`, you may decode this JSON string into base64 encoding.
...
+      {
+        "info" : {
+          "native_token": {
+            "denom": "uluna"
+          }
+        },
+        "amount": "10"
+      }
+    ], // optional
+    "deadline": 1686903051 // optional
```
Comment (jbamlee) [inline content/en/docs/how-to/frequently-used-messages.md]: add `unix epoch` for the consistency

File: content/en/docs/how-to/frequently-used-messages.md [hunk 2]
```diff
@@ -37,7 +37,9 @@ If you are trying to provide cw20 token, increase your allowance first. [Execute
         },
         "amount": "10"
       }
-    ]
+    ],
+    "deadline": 1686903051, // optional
```
Comment (jbamlee) [inline content/en/docs/how-to/frequently-used-messages.md]: Nit: add `unix epoch` for the consistency

**Review by jbamlee — APPROVED** [review]

File: content/en/docs/how-to/frequently-used-messages.md (hunk 1 above)
Comment (jhlee-young) [reply content/en/docs/how-to/frequently-used-messages.md]: [3566b67](https://github.com/terraswap/docs/pull/3/commits/3566b67982bf123d4767c6164171235b82a3b112)

File: content/en/docs/how-to/frequently-used-messages.md (hunk 2 above)
Comment (jhlee-young) [reply content/en/docs/how-to/frequently-used-messages.md]: [3566b67](https://github.com/terraswap/docs/pull/3/commits/3566b67982bf123d4767c6164171235b82a3b112)

**Pull request merged**